src/bp-activity/bp-activity-template.php

```diff
@@ -4184,26 +4160,29 @@ function bp_activity_entry_css_class() {
 	 *
 	 * @return string The activity entry item's CSS class.
 	 */
-	function bp_get_activity_entry_css_class() {
-		global $activities_template;
+function bp_get_activity_entry_css_class() {
+	global $activities_template;
 
-		$class = '';
+	$class = '';
 
-		if ( bp_is_active( 'media' )  ) {
-			// $document_activity = bp_activity_get_meta( bp_get_activity_id(), 'bp_document_activity', true );
-			$document_ids = bp_activity_get_meta( bp_get_activity_id(), 'bp_document_ids', true );
-			if( ! empty( $document_ids ) ){
-				$class .= ' documemt-activity';
-			}
+	if ( bp_is_active( 'media' ) ) {
+		$document_ids = bp_activity_get_meta( bp_get_activity_id(), 'bp_document_ids', true );
+		if ( ! empty( $document_ids ) ) {
+			$class .= ' documemt-activity';
 		}
 
-		/**
-		 * Filters the determined classes to add to the HTML element.
-		 *
-		 * @since BuddyBoss 1.5.0
-		 *
-		 * @param string $value Classes to be added to the HTML element.
-		 */
-		return apply_filters( 'bp_get_activity_entry_css_class', $class );
+		$media_ids = bp_activity_get_meta( bp_get_activity_id(), 'bp_media_ids', true );
+		if ( ! empty( $media_ids ) ) {
+			$class .= ' media-activity-wrap';
+		}
 	}
 
+	/**
+	 * Filters the determined classes to add to the HTML element.
+	 *
+	 * @since BuddyBoss 1.5.0
+	 *
+	 * @param string $value Classes to be added to the HTML element.
+	 */
+	return apply_filters( 'bp_get_activity_entry_css_class', $class );
```

Comment:
@chetansatasiya use this filter in media and document component codebase and add classes there.

Reply:
👍